src/test/java/edu/ksu/canvas/attendance/services/AttendanceSummaryCSVServiceUTest.java

```diff
@@ -37,8 +40,11 @@
     private final int ARBITRARY_STUDENT_MISSED_2 = 5;
     private final int ARBITRARY_STUDENT_TOT_MADEUP_2 = 0;
     private final int ARBITRARY_STUDENT_TOBE_MADEUP_2 = 6;
+    private final int ARBITRARY_STUDENT_EXCUSED_2 = 1;
+    private final int ARBITRARY_STUDENT_PRESENT_2 = 3;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "ARBITRARY_STUDENT_PRESENT_2" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00116)